termsaverlib/screen/img2ascii.py

```diff
@@ -0,0 +1,188 @@
+###############################################################################
+#
+# file:     img2ascii.py
+#
+# Purpose:  refer to module documentation for details
+#
+# Note:     This file is part of Termsaver application, and should not be used
+#           or executed separately.
+#
+###############################################################################
+#
+# Copyright 2012 Termsaver
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+###############################################################################
+"""
+A screen to read source code files (or any other) in a  typing writer animation
+See additional information in the class itself.
+
+The helper class available here is:
+
+    * `Img2Ascii`
+"""
+from termsaverlib.screen.base.imagereader import ImageReaderBase
+from termsaverlib import constants
+from termsaverlib.i18n import _
+
+import os
+
+class Img2Ascii(ImageReaderBase):
```

Comment:
Can the ImageReaderBase be used in any other way that is not Img2Ascii? If the answer is no, I'd rather combine this into one, and remove from base. If you see any way it can be reused, then happy to keep as is.